pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1889,7 +1889,6 @@
     "System.IO.Compression.Brotli": {
       "InboxOn": {
         "netcoreapp2.1": "4.2.1.0",
```

Comment:
I don't know how this file is parsed but normally trailing commas are not allowed in JSON.

Reply:
Thanks, fixed.